main.py

```
@@ -70,5 +94,5 @@ async def predict_yield(data: PredictRequest):
         predicted_yield = model.predict(df)[0]
         return {"predicted_ExpYield": float(predicted_yield)}
     except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)